modules/react/common/lib/parts/_brand-assets.ts

```diff
@@ -1,11 +1,11 @@
 const CDN_URI = '//design.workday.com/images/';
 
-export const dubLogoBlue = `<img src="${CDN_URI}ck-dub-logo-blue.svg">`;
+export const dubLogoBlue = `<img src="${CDN_URI}ck-dub-logo-blue.svg" aria-label="">`;
```

Comment:
I think an `alt` attribute should be present and not an `aria-label`. And it shouldn't be empty. It should say "Workday". If any other word (like "logo") is included, we may have to translate it.

```suggestion
export const dubLogoBlue = `<img src="${CDN_URI}ck-dub-logo-blue.svg" alt="Workday">`;
```

Reply:
Github didn't allow me to span a suggestion across all lines

Reply:
The W3C article linked mentioned an empty alt attribute and not an empty aria-label attribute. I'm asking if the alt value should be empty or "Workday".

Reply:
@NicholasBoll You're correct, instead of an empty aria-label it should be an empty alt property. Empty alt tells the browser that the image is decorative, if we set alt to Workday, screen readers would read the alt value as an image description, which is what we are trying to avoid. I'll fix this now.